keymaps/latex.json

```diff
@@ -1,7 +1,11 @@
 {
+  "atom-text-editor": {
+    "ctrl-alt-l": "latex:toggle-log"
+  },
   "atom-text-editor[data-grammar~=\"latex\"]": {
     "ctrl-alt-b": "latex:build",
     "ctrl-alt-s": "latex:sync",
-    "ctrl-alt-c": "latex:clean"
+    "ctrl-alt-c": "latex:clean",
+    "ctrl-alt-e": "latex:sync-log"
```

Comment:
Can we avoid needing it by default? Users can still create whatever keybinding they want for the command, but perhaps we can circumvent having one by default by introducing e.g. a context menu item for this? That's potentially more discoverable as well.